src/components/TabBar/TabBar.css

```diff
@@ -2,18 +2,49 @@
 
 :export { inkBarColor: var(--primaryLinkColor) }
 
+.container {
+  position: relative;
+  @media only screen and (max-width : 1250px) {
+    height: 48px;
+    overflow-y: hidden;
+  }
+
+  @media only screen and (max-width: 960px) {
+    margin: 0 -25px;
```

Comment:
You can remove this since I add mq on my branch https://github.com/odota/ui/blob/68adfa66991c222218ccdf52ffcb7bb7368a4d6b/src/components/App/App.css#L23